.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         components: rustfmt
 
     - name: cargo winrt
-      run: cargo install --path ./crates/cargo-winrt
+      run: cargo install --path ./crates/cargo
```

Comment:
FWIW I don't love calling this directory `cargo`. The crate is called `cargo-winrt`, and while it's marginally longer to have to type the additional 6 characters, it's much clearer what the directory contains for the uninitiated.

Reply:
Just trying to bring some consistency. The crates folder contains a number of crates that all include `winrt` in their names but this was the only one that spelled it out. Same of the other comments below. Some crates used `-` and others `_`.

Reply:
The directory names and the crate names don't _need_ to align, but it can help. I would personally prefer that the directories in `crates` were all named exactly like the crates themselves. It's unfortunate about the `-` vs `_`. We can rename crates to be consistent, but it's a bit of a shame we'll have to deprecate the old crates which use `_`. This isn't a huge deal though since they're just implementation details...